src/Analysis/Ast/Impl/Specializations/BuiltinsSpecializations.cs

```diff
@@ -175,5 +175,17 @@ public static IMember List(IPythonInterpreter interpreter, IPythonFunctionOverlo
 
             return o?.GetPythonType().GetMember(name) ?? def;
         }
+
+        public static IMember Enumerate(IPythonModule module, IPythonFunctionOverload overload, IArgumentSet argSet, IndexSpan indexSpan) {
+            var args = argSet.Values<IMember>();
+            if (args.Count > 0) {
+                var iterator = (args[0] as IPythonIterable)?.GetIterator();
+                var itemType = (iterator?.Next ?? module.Interpreter.UnknownType).GetPythonType();
+                var intType = module.Interpreter.GetBuiltinType(BuiltinTypeId.Int);
+                var content = new[] {new PythonInstance(intType), itemType.CreateInstance(ArgumentSet.WithoutContext)};
```

Comment:
`intType.CreateInstance(ArgumentSet.WithoutContext)` maybe? I think we've been avoiding doing `new PythonInstance` directly in favor of always calling `CreateInstance`.

Reply:
Sure.